bar.py

```diff
@@ -13,14 +13,14 @@ def bar_one(x, y):
 
 def bar_two(z, q):
     bat.do_z(z)
-    if name:
-        bat.do_q(q)
-    else:
-        bat.do_q(q + 7)
+    bat.do_q(q + name)
```

Comment:
**mike bayer** (zzzeek) wrote:

review comment.   code example:

    from foo import bar
    print(x + y)

like that.